docs/howto/deploy/azure.md

```diff
@@ -1,12 +1,13 @@
 ---
 layout: default
 title: Azure
-parent: Deploy and Setup lakeFS
+grand_parent: How-To
+parent: Install lakeFS
```

Comment:
I see your point. However we need some structure to the docs and the Diátaxis approach I think works well. It will take a bit of getting used to, and for some things there are exceptions (e.g. I would not hide Quickstart - at least for now, unless we end up with lots of `Tutorials` under which to nest it). But installation is a "how do I do this thing", so does make sense under How To. 

Regarding the ordering, have a look at https://github.com/treeverse/lakeFS/pull/6257 and specifically https://treeverse-lakefs-preview-pr-6257.surge.sh/howto/ to see if this makes more sense.

Reply:
Like you said, there are exceptions to taking this approach and I think this should be one. I sampled some sites using Diátaxis and they all had some kind of "Getting started" category, including all the ways to install/buy the product.

Reply:
How about if I set the `nav_order` to make it the first item under How To?